src/Hedgehog/Gen.fs

```diff
@@ -3,12 +3,17 @@
 open System
 open Hedgehog.Numeric
 
+
```

Comment:
What's the purpose of that extra line?

Reply:
Absolutely none. I'll remove that rogue edit. It was caused by my adding `fsharpDouble` there first before realizing it had to be moved, and then I forgot to check the diff before I committed.